cloudformation.json

```diff
@@ -58,6 +58,24 @@
                 "Roles": [{"Ref": "RootRole" }]
             }
         },
+        "DynamoDBPolicy": {
+            "Type": "AWS::IAM::Policy",
+            "Properties": {
+                "PolicyName": "DynamoDBPolicy",
+                "PolicyDocument": {
+                    "Statement": [{
+                        "Effect": "Allow",
+                        "Action": [ "dynamodb:*" ],
+                        "Resource": [
+                                { "Fn::Join": [ "-", [ "arn:aws:dynamodb:*:308506855511:table/content-source", { "Ref": "Stage" } ] ] },
```

Comment:
We should include a `floodgate-` prefix in all table names. Also, you could write this with a wildcard like

```
{ "Fn::Join": [ "-", [ "arn:aws:dynamodb:*:308506855511:table/floodgate-*", { "Ref": "Stage" } ] ] }
```

so it doesn't break when you wan't to add more tables.